common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -906,47 +905,79 @@ div.problem {
 // +Problem - Actions
 // ====================
 div.problem .action {
-  margin-top: $baseline;
+  @include margin($baseline 0 $baseline 0);
```

Comment:
nit: I believe `($baseline 0 $baseline 0)` can be expressed as just `($baseline 0)` (although I don't know if the internal implementation of the `margin()` sass mixin plays by the same rules as plain css `margin` prop is so this could be wrong)